h5netcdf/tests/test_h5netcdf.py

```diff
@@ -1387,3 +1401,32 @@ def test_no_circular_references(tmp_local_netcdf):
     gc.collect()
     with h5netcdf.File(tmp_local_netcdf, "r") as ds:
         assert len(gc.get_referrers(ds)) == 1
+
+
+def test_creation_with_h5netcdf_edit_with_netcdf4(tmp_local_netcdf):
+    # In version 0.12.0, the wrong file creation attributes were used
```

Comment:
A test of the current known-fail would be good. So we know, when netcdf-c/netcdf4-python has implemented the new feature of being track_order agnostic.

Reply:
Added one.

Reply:
This is now rather hard to do because of the new error behavior. I've created an issue in this repo instead
https://github.com/h5netcdf/h5netcdf/issues/130

Reply:
We could just utilize `h5py` instead of `h5netcdf` to create that feature.

Reply:
The tracking issue is fine anyway.

Reply:
> We could just utilize h5py instead of h5netcdf to create that feature.

I'm not sure what this means.

Reply:
Something along these lines:

```python
import h5py
import netCDF4
with h5py.File('test.nc', 'w', track_order=True) as f:
    pass
with netCDF4.Dataset("test.nc", mode="a") as f:
    pass
```

Reply:
Very smart. added. thanks!